mesh/src/main/java/no/nordicsemi/android/mesh/sensorutils/DeviceProperty.java

```diff
@@ -1056,6 +1056,9 @@ public static String getPropertyName(final DeviceProperty deviceProperty) {
             case LUMINAIRE_POWER_AT_MINIMUM_DIM_LEVEL:
             case PRESENT_DEVICE_INPUT_POWER:
                 return new Power(data, offset, 3);
+            case PRESENT_INPUT_CURRENT:
+            case PRESENT_OUTPUT_CURRENT:
+                return new ElectricCurrent(data, offset, 2);
```

Comment:
It's rather odd to pass the hardcoded length here instead of the actual length parameter getCharacteristic()? What if data doesn't contain exactly 3 bytes?

Reply:
Length is number of bytes used to encode the characteristic, not length of data.